mbed-os.lib

```diff
@@ -1 +1 @@
-https://github.com/ARMmbed/mbed-os/#e1bea44212b8275f7d8ce7253e758c2e25c57482
```

Comment:
Why changes to Mbed OS? Remove this.

Reply:
I never intended this file to be in the commit, not sure how it crept in, I'll be carefull next time, thanks.